controllers/router.js

```diff
@@ -15,33 +14,38 @@ var house = {
 };
 
 // Map routes to controller functions
-module.exports = function(app) {
+module.exports = function () {
+    // Create an authenticated Twilio API client
+    var client = new Twilio(config.accountSid, config.authToken);
```

Comment:
I was trying not to make too many changes in regards to const/var because then I would have to do it everywhere. 😉